CHANGES.md

```diff
@@ -34,6 +34,7 @@ Change Log
 * Updated `requirejs` from 2.1.9 to 2.1.20. [#3107](https://github.com/AnalyticalGraphicsInc/cesium/pull/3107)
 * Updated `almond` from 0.2.6 to 0.3.1. [#3107](https://github.com/AnalyticalGraphicsInc/cesium/pull/3107)
 * Made `TileMapServiceImageryProvider` and `CesiumTerrainProvider` work properly when the provided base url contains query parameters and fragments.
+* Allow custom geoJson description function.
```

Comment:
This needs to be moved to a `1.16` section.  I would also be more specific, something like:

`GeoJsonDataSource.load` now takes an optional `describe` function for generating feature descriptions. #3140